operator/operator_test.go

```diff
@@ -183,9 +183,7 @@ func (e *environment) WaitPhase() {
 
 func (e *environment) Shutdown() {
 	if e.cmd != nil {
-		if err := e.cmd.Process.Kill(); err != nil {
-			log.Warn("Could not kill process:", err)
-		}
+		e.cmd.Process.Kill() // nolint: errcheck
```

Comment:
What's the linter's problem here? Wouldn't it be better to log the error?

Reply:
It collided with a change from Matthias, i will opt for the log.